src/state/profile/hooks.ts

```diff
@@ -18,34 +17,14 @@ export const useFetchProfile = () => {
   }, [account, dispatch])
 }
 
-export const useProfileForAddress = (address: string) => {
```

Comment:
This is similar hook I made recently, yeet 🗑️

Reply:
Link to yours Cheems? Should yours now live in this file?

Reply:
What do you mean? This is the hook I used before I learned that Hachi has one already. Was used on individual NFT page only, now replaced with Hachi's hook

Reply:
Ah sorry - I thought this was Hachi's hook that you were trashing in favour of yours. All good 👍